packages/fakta-medisinsk-vilkår/mock/handlers.ts

```diff
@@ -17,8 +19,22 @@ import mockedToOmsorgspersonerVurderingsoversikt from './mocked-data/mockedToOms
 
 let mockedNyeDokumenter = [...mockedNyeDokumenterList];
 
+type EndreVurderingRequestBody = NyVurderingsversjon & {
```

Comment:
burde ikke typene under eksistere fra før av? hvis ikke burde de vel legges inn der vi sender requestet i koden også?

Reply:
Usikker på om mocken her er up to date med faktiske data fra backend

Reply:
I praksis så tror jeg disse feltene jeg utvider med (endretAv og endretTidspunkt) settes av backend